Scripts/application.py

```diff
@@ -0,0 +1,58 @@
+import sqlite3
+import json
+from typing import Dict
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+    
+@app.route('/api/v1/person')
+def return_people():
+    # returns a json of all people
+    with sqlite3.connect('../db/allYourSantaAreBelongToUs.db') as connection:
+        cursor = connection.cursor()
+        #get all of the people
+        cursor.execute('SELECT * FROM people')
+        raw_data: List = cursor.fetchall()
+        data: Dict = {}
+        for person in raw_data:
+            data[person[0]] = {"email": person[1], "assignment ID": person[2]}
+        json_data = json.dumps(data)
+    return json_data
+    
+@app.route('/api/v1/person/<person_id>')
+def return_person(person_id):

```

Comment:
can you specify the type for person_id here?

Reply:
Yes.

Reply:
When I pull this parameter from the decorator, is it a string or an int?

Reply:
https://lmgtfy.com/?q=python+check+if+variable+is+int+or+string

Reply:
🖕